lib/validation/validators/slackChannelName.ts

```diff
@@ -0,0 +1,25 @@
+import { isString } from '../../support/Utils'
+
+export const SLACK_CHANNEL_NAME_MAX_LENGTH = 80 as const
+const SLACK_CHANNEL_NAME_INVALID_SUMBOLS = /[ ,.!@#$%^&*?(){}<>=+|・'`~/\\\[\]\"]/
```

Comment:
I will explain this point in another place.

Reply:
Slightly better version for validity - `/^[\p{L}\p{M}\p{N}_-]{1,80}$/u` (some of the marks (`\p{M}`) are not valid unfortunately, so it will allow some invalid characters).

Reply:
@kiaking 

What do you think?

Reply:
Sure why not! As long as test is passing. It should be fine 👀

Reply:
Refined validator based on the regular expression: 45ddb11

JFYI, here is a ref about the regular expression: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape

Note that discussion on implementation is still ongoing at https://github.com/globalbrain/sefirot/pull/488#discussion_r1513648136

@brc-dd What do you think about the conversation above?